### ISSUE TITLE
Show build output in new TAB, not in panel 

### ISSUE THREAD

**Issue by quantumgolem** [SublimeText/LaTeXTools#1484]
I work on a laptop, so when the build output appears below my code, it becomes really cramped. I would much rather have the build output inside a new tab instead of the build output area. 

One way to achieve this could be to write the build output to disk, which would allow me to open it in a new tab. Then I could always switch to that tab if I ever needed to check what was going on. 

Another way I thought I could solve it was to install buildview in Sublime Text and the build output would appear in a new tab, as described over [here](https://stackoverflow.com/questions/23444125/how-to-change-position-of-output-panel-in-sublime-text). However, after installation unfortunately it doesn't seem to do anything :/.

**Comment by sn0wyfall** [comment]
I tried investigating buildview a bit more, but haven't been able to figure it out. However, it might be able to do what I'm suggesting. Does anyone have any tips on how to configure it? When I build with it installed, it keeps on building in the same panel over my current document. Do I need to do anything, like change my keyboard shortcuts, for it to start working?

**Comment by ig0774** [comment]
I don’t think LaTeXTools is compatible with that plugin. We write our build information to a different output panel than the default build output panel (“exec”), which allows us to support some nice features. The easiest thing to do would be to update `buildview` to handle a configurable output panel (I think the one we use is called “latextools”).

**Comment by sn0wyfall** [comment]
Oh, I see, thanks for the tip. So there's no way currently to achieve this? Perhaps a cache or log file in the Sublime Text working directory which contains the information? Thanks!

**Comment by ig0774** [comment]
I don’t think we cache the information per se, but the information we display is all extracted from the LaTeX log and [our script that does that parsing](https://github.com/SublimeText/LaTeXTools/blob/master/parseTeXlog.py) can be run outside of LaTeXTools (though you’d want to set the `print_debug` and `interactive` variables [here](https://github.com/SublimeText/LaTeXTools/blob/45376f12449e32c4ad769f5baa0a75c1fa72194d/parseTeXlog.py#L751) to `False`).

Basically you could get similar output by running: `python parseTeXlog.py texfile.log`. Of course, that wouldn’t have the clickable links and colour-coding.

**Comment by sn0wyfall** [comment]
Thanks a lot! That's helpful to know. I guess this issue isn't important enough for the developers to recode it so that it will work for buildview and/or will show up in a new tab. 

I have two questions then:

1. Is there any way I could hide the view if there are no errors? It shows up every time I render my file while gets annoying since I have to dismiss it all the time. Would I do this by disabling `print_debug`? But this would disable it even if there are errors, I presume? 

2. If there isn't, maybe the best option is to keep it minimized and then expand it when I need it. But to do that, I'd need to know when there are errors/warnings. Is it possible to add that information in the first line? Then I could keep it minimized and I'd be able to see when I need to look at the log. For example, the first line could say "Build failed. 8 warnings and 3 errors." That would be really useful. Or maybe I could modify it to be like that myself? Would I modify [this file](https://github.com/SublimeText/LaTeXTools/blob/master/parseTeXlog.py) to achieve what I want?
